### PR TITLE
MLOps definition - add examples for add't clarity

### PR DESCRIPTION
**Pull request by epec254**
I agree with the statement /*The extension of the DevOps methodology to include Machine Learning and Data Science Assets and the interdependencies between those Assets as first class citizens within the DevOps ecology.*/ and the broad framing of this section :-)

I added some specific examples of "ML Assets" that I think could help increase the clarity of what is needed.  I seeded a high level list of assets that I felt was representative enough to get the point across, but I would welcome feedback if there are additional assets worth calling out or if we should go to the next level of depth.

### CODE REVIEW

File: roadmap/2022/MLOpsRoadmap2022.md [hunk 1]
```diff
@@ -40,21 +40,47 @@ Almog Baku, Rimoto <almog.baku@gmail.com>
...
+> *The extension of the DevOps methodology to include Machine Learning and Data Science Assets and the interdependencies between those Assets as first class citizens within the DevOps ecology.*
 
+Some ML & DS Assets require new extensions to DevOps practices:
+* Features [1]
+* Trained Models [2]
+
+Some ML & DS Assets can be handled by existing software DevOps practices, but must be linkable to the other ML Assets
+* Model Servers [3]
+* Model Training Pipelines [4]
+
```
Comment (AlmogBaku) [inline roadmap/2022/MLOpsRoadmap2022.md:60]: I suggest we'll mention that these new resources should obviously be linked to lower-level DevOps resources such as computing, storage, etc.

File: roadmap/2022/MLOpsRoadmap2022.md [hunk 2]
```diff
@@ -40,21 +40,47 @@ Almog Baku, Rimoto <almog.baku@gmail.com>
...
+MLOps must have first-class integration with existing DataOps processes since well-performing machine learning relies on complete, consistent, semantically valid, correct, timely, and unbiased **Input Data**.  Without this clean, accurate data, ML solutions can be problematic or lead to flawed outcomes that exacerbate biases.
```
Comment (AlmogBaku) [inline roadmap/2022/MLOpsRoadmap2022.md:61]: As an extension for DevOps, MLOps should also have integration with existing DevOps tooling

File: roadmap/2022/MLOpsRoadmap2022.md [hunk 3]
```diff
@@ -40,21 +40,47 @@ Almog Baku, Rimoto <almog.baku@gmail.com>
...
+Accordingly, we believe a better definition of MLOps is:
+> *The extension of the DevOps methodology to include Machine Learning and Data Science Assets and the interdependencies between those Assets as first class citizens within the DevOps ecology.*
 
+Some ML & DS Assets require new extensions to DevOps practices:
+* Features [1]
+* Trained Models [2]
+
+Some ML & DS Assets can be handled by existing software DevOps practices, but must be linkable to the other ML Assets
+* Model Servers [3]
+* Model Training Pipelines [4]
```
Comment (AlmogBaku) [inline roadmap/2022/MLOpsRoadmap2022.md:59]: I think that pipelines and model servers are not assets but procedures, and their artifacts are the  assets

File: roadmap/2022/MLOpsRoadmap2022.md [hunk 4]
```diff
@@ -40,21 +40,47 @@ Almog Baku, Rimoto <almog.baku@gmail.com>
...
 ### What is MLOps not?
 
 It sometimes helps to consider what anti-patterns exist around a concept in order to better understand it.
 
-For example, MLOps is not "putting Jupyter Notebooks into production environments".
+**1. MLOps is not "putting Jupyter Notebooks into production environments".**
+
+Rapid Application Development (RAD) tools like Jupyter Notebooks can be extremely useful, both in classroom environments and in exploring problem spaces to understand potential approaches to mathematical problems. However, like all RAD tools, they achieve the rapid element of their name by trading off other key non-functional requirements like maintainability, testability and scalability.  
+
+**2. MLOps is not a purely technical or infrastructure solution**
```
Comment (AlmogBaku) [inline roadmap/2022/MLOpsRoadmap2022.md:81]: I don't agree with this statement. IMO:
> MLOps != ML Infra

As DevOps != Dev Infra (i.e. Kafka and K8s are not sitting under the same category). Although some VCs are confusing between these two subjects, they're not the same.

File: roadmap/2022/MLOpsRoadmap2022.md [hunk 5]
```diff
@@ -40,21 +40,47 @@ Almog Baku, Rimoto <almog.baku@gmail.com>
...
 
-RAD tools like Jupyter Notebooks can be extremely useful, both in classroom environments and in exploring problem spaces to understand potential approaches to mathematical problems. However, like all Rapid Application Development tools, they achieve the rapid element of their name by trading off other key non-functional requirements like maintainability, testability and scalability.
+MLOps, similar to DevOps, includes a combination of enabling technology and infrastructure, but also requires people-based processes to be successfully implemented.
```
Comment (AlmogBaku) [inline roadmap/2022/MLOpsRoadmap2022.md:83]: Should be "MLOps, *as an extension* to DevOps, ...."

File: roadmap/2022/MLOpsRoadmap2022.md [hunk 6]
```diff
@@ -40,21 +40,47 @@ Almog Baku, Rimoto <almog.baku@gmail.com>
 
 ### What is MLOps?
 
-MLOps could be narrowly defined as "the ability to apply DevOps principles to Machine Learning applications" however as we shall see shortly, this narrow definition misses the true value of MLOps to the customer. Instead, we define MLOps as “the extension of the DevOps methodology to include Machine Learning and Data Science assets as first class citizens within the DevOps ecology”.
+MLOps is often narrowly defined as:
```
Comment (AlmogBaku) [inline roadmap/2022/MLOpsRoadmap2022.md:43]: Love this edit, much simpler IMO

**Review by AlmogBaku — CHANGES_REQUESTED** [review]
Thank you for your contribution Eric!

I added a few notes to it.

File: roadmap/2022/MLOpsRoadmap2022.md (hunk 4 above)
Comment (AlmogBaku) [reply roadmap/2022/MLOpsRoadmap2022.md:81]: hmm... maybe I understood it wrong? if so - the title is a bit confusing

Comment (epec254) [reply roadmap/2022/MLOpsRoadmap2022.md:81]: Are you referring to line 81 or line 78-79?

File: roadmap/2022/MLOpsRoadmap2022.md (hunk 2 above)
Comment (epec254) [reply roadmap/2022/MLOpsRoadmap2022.md:61]: I agree, I think this is covered by "/The extension of the DevOps methodology to include Machine Learning and Data Science Assets and the interdependencies between those Assets as first class citizens within the DevOps ecology./ "

earlier in the doc.  WDYT?

File: roadmap/2022/MLOpsRoadmap2022.md (hunk 3 above)
Comment (epec254) [reply roadmap/2022/MLOpsRoadmap2022.md:59]: Curious to hear more on why they are procedures?  I was thinking the code/docker containers/etc + the link to the needed (or geneated) Features + Trained Models are included in each of these as the assets...

File: roadmap/2022/MLOpsRoadmap2022.md (hunk 5 above)
Comment (epec254) [reply roadmap/2022/MLOpsRoadmap2022.md:83]:
```suggestion
MLOps, as an extension to DevOps, includes a combination of enabling technology and infrastructure, but also requires people-based processes to be successfully implemented.

```

**Comment by tdcox** [comment]
Thanks. It’s not part of the remit of this document to define ML assets or techniques, since these very widely and change significantly over time. We are really looking for contributions that identify specific difficult challenges within the process of managing ML assets as part of a product lifecycle.

**Comment by AlmogBaku** [comment]
@tdcox  to what part are you referring? Specifying the list of assets? That might refine the problem definition

**Pull request closed without merging**